app/components/Navigation.js

```diff
@@ -59,7 +59,7 @@ function Navigation (props) {
       }
 
       return (
-        h(NavLink, {
+        h(Link, {
```

Comment:
NavLink ensures the link can be highlighting, this is in the navigation column so should stay using NavLink